content/applications/inventory_and_mrp/manufacturing/management/sub_assemblies.rst

```diff
@@ -2,59 +2,70 @@
-products to the Bill of Material. 
+Once the top-level product is configured, click the :guilabel:`Bill of Materials` smart button on
+the product form, then click :guilabel:`Create` to make a :abbr:`BoM (Bill of Materials)` for the
+top-level product. Then, simply add the semi-finished products to this :abbr:`BoM (Bill of
+Materials)`, along with any other necessary components.
 
-.. image:: sub_assemblies/sf_2.png
-    :align: center
+.. image:: sub_assemblies/custom_computer_bom.png
+   :align: center
+   :alt: A bill of materials for a top-level product, containing a subassembly component.
 
-Manage your production planning 
-=================================
+Manage production planning
+==========================
 
-There are several methods to manage the triggering of the various manufacturing orders. 
+There are several methods to manage manufacturing order automation for products with multilevel
+:abbr:`BoMs (Bills of Materials)`.
 
-If every time a manufacturing order is confirmed for the main product, you'd like one for 
-the semi-finished products as well, you have two options: 
+.. note:: 
+    Semi-finished products are specifically used to manage manufacturable products with multilevel
+    BoMs. If a BoM is being created simply to organize components or bundle sellable products,
+    using :doc:`Kits <kit_shipping>` is the more appropriate option.
 
-Option 1 : Create re-ordering rules for the semi-finished products, with both the minimum 
-and maximum desired stock quantities at 0. 
+To automatically trigger manufacturing orders for semi-finished products after confirming a
+manufacturing order for the main product, there are two options:
 
-.. image:: sub_assemblies/sf_3.png
-    :align: center
+**Option 1 (recommended):** Create *Reordering Rules* for the semi-finished products and set both
+the minimum and maximum desired stock quantities to '0'. To review how to set up reordering rules,
+refer to :doc:`this page <../../../inventory_and_mrp/purchase/products/reordering>`.
 
-Option 2 : Use the Replenish on Order (MTO) route on the semi-finished product, as well as 
-the manufacturing one. 
+**Option 2:** Activate the *Replenish on Order (MTO)* and *Manufacture* routes under the
+:guilabel:`Inventory` tab of the semi-finished product’s product form.
 
-Note that Option 1 is usually recommended over Option 2 as it is more flexible. The MTO route 
-creates a unique link between the semi-finished and the top level product, whereas the 
-re-ordering rule allows you to simply unreserve that production from the top level product
-production and redirect it to another, more pressing demand, for example. 
+Option 1 is more flexible than Option 2 and is therefore recommended. Reordering rules do not
+directly link demand to replenishment, and therefore allow stocks to be unreserved and redirected
+to other orders, if necessary. The Replenish on Order (MTO) route creates a unique link between the
```

Comment:
Suggestion since we're _referencing_ something on the UI but not in an instructional way. 
Feel free to ignore if you disagree.

```suggestion
to other orders, if necessary. The *Replenish on Order (MTO)* route creates a unique link between the
```